{
	"id": "oval:com.oracle.elsa:def:20203658",
	"advisories": [
		{
			"id": "oval:com.oracle.elsa:def:20203658",
			"title": "\nELSA-2020-3658:  librepo security update (IMPORTANT)\n",
			"description": "\n[1.11.0-3]\n- Validate paths read from repomd.xml (RhBug:1866498)\n",
			"severity": [
				{
					"type": "vendor",
					"source": "linux.oracle.com/security",
					"vendor": "IMPORTANT"
				}
			],
			"references": [
				{
					"source": "linux.oracle.com/security",
					"url": "https://linux.oracle.com/cve/CVE-2020-14352.html"
				},
				{
					"source": "linux.oracle.com/security",
					"url": "https://linux.oracle.com/errata/ELSA-2020-3658.html"
				}
			],
			"published": "2020-09-09T00:00:00Z"
		}
	],
	"vulnerabilities": [
		{
			"id": "CVE-2020-14352",
			"references": [
				{
					"source": "linux.oracle.com/security",
					"url": "https://linux.oracle.com/cve/CVE-2020-14352.html"
				}
			]
		}
	],
	"detection": [
		{
			"ecosystem": "oracle:8",
			"criteria": {
				"operator": "OR",
				"criterions": [
					{
						"vulnerable": true,
						"package": {
							"name": "librepo",
							"architectures": [
								"aarch64",
								"x86_64"
							]
						},
						"affected": {
							"type": "rpm",
							"range": [
								{
									"lt": "0:1.11.0-3.el8_2"
								}
							],
							"fixed": [
								"0:1.11.0-3.el8_2"
							]
						}
					},
					{
						"vulnerable": true,
						"package": {
							"name": "python3-librepo",
							"architectures": [
								"aarch64",
								"x86_64"
							]
						},
						"affected": {
							"type": "rpm",
							"range": [
								{
									"lt": "0:1.11.0-3.el8_2"
								}
							],
							"fixed": [
								"0:1.11.0-3.el8_2"
							]
						}
					}
				]
			}
		}
	],
	"data_source": "oracle"
}
